internal/io/sql/reader.go

```diff
@@ -0,0 +1,96 @@
+package sql
+
+import (
+	"database/sql"
+
+	"github.com/tobgu/qframe/types"
+)
+
+// args returns a slice of Scanner for the SQL
+// package. TODO: These pointer allocations seem
+// unnecessary but I'm unsure how else to do it.
+func args(cols []*Column) (ptrs []interface{}) {
```

Comment:
I've not used named returned types anywhere else in this code base.

I would rather see an explicit allocation of `cols` with a set size by calling `make`.